src/components/DropzoneArea.js

```diff
@@ -507,6 +510,15 @@ DropzoneArea.propTypes = {
      * @param {number} maxFileSize The `maxFileSize` prop currently set for the component
      */
     getDropRejectMessage: PropTypes.func,
+    /**
+     * A function which determines which icon to display for a file preview.
+     *
+     * Default*: If its an image then displays a preview the image, otherwise it will display an attachment icon
```

Comment:
Check `Default` formatting, it seems that a `*` is missing to make it *italic*

Reply:
Oh ok, will check this